components/director/pkg/accessstrategy/cmp_mtls_access_strategy.go

```diff
@@ -19,7 +19,7 @@ type HTTPRoundTripper interface {
 	GetTransport() *http.Transport
 }
 
-const tenantHeader = "tenant"
+const tenantHeader = "Tenant_Id"
```

Comment:
I approved the PR but I was wondering what is the motivation behind this change? Because until now we were using the  `tenant` key and the other side(whoever is it) has known to extract the value from it but now we're changing the header key, so I guess the other participant should adapt that from his side as well?

Reply:
the other side is aware of that change